tools/java/runfiles/testing/RunfilesTest.java

```diff
@@ -249,10 +250,16 @@ public void testManifestBasedRlocation() throws Exception {
         new MockFile(
             ImmutableList.of(
                 "Foo/runfile1 C:/Actual Path\\runfile1",
-                "Foo/Bar/runfile2 D:\\the path\\run file 2.txt"))) {
+                "Foo/Bar/runfile2 D:\\the path\\run file 2.txt",
+                "Foo/Bar/Dir E:\\Actual Path\\Directory"))) {
       Runfiles r = Runfiles.createManifestBasedForTesting(mf.path.toString());
       assertThat(r.rlocation("Foo/runfile1")).isEqualTo("C:/Actual Path\\runfile1");
       assertThat(r.rlocation("Foo/Bar/runfile2")).isEqualTo("D:\\the path\\run file 2.txt");
```

Comment:
This existing test case seems to go against the specification of `rlocation` in https://docs.google.com/document/d/e/2PACX-1vSDIrFnFvEYhKsCMdGdD40wZRBX3m3aZ5HhVj4CtHPmiXKDCxioTUbYsDydjKtFDAzER5eg7OjJWs3V/pub: It demands the return value to be an absolute path that uses `\` as the directory separator, whereas the design doc demands all paths to use `/` as path separators. 

Depending on what is correct, I would replace the uses of OS-specific path joining logic with simple concenations with `/` as the separator.

Reply:
I think the documentation is correct, in reality, only `/` is used in the runfiles manifest file, feel free to fix this test case! Let's also use `/` for this case to be consistent and less error prone.

Reply:
I'm using `/` as a separator in all cases. I will leave the test case I mentioned as is: I assume it exists more to show that manifest entries are left as is and not modified.